### PR TITLE
New tutorial "How to transfer files using scp command" 

### PR DESCRIPTION
**Pull request by pixia1234**
I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: pixia1234 [pixia2005@gmail.com](mailto:pixia2005@gmail.com)

### CODE REVIEW

File: tutorials/howto-transfer-files-using-scp/01.en.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/howto-transfer-files-using-scp"
+slug: "howto-transfer-files-using-scp"
+date: "2024-01-11"
+title: "How to transfer files using the scp command"
+short_description: "Use scp to transfer files easily between your computer and servers!"
+tags: ["SCP", "SSH"]
+author: "CY Wei"
```
Comment (svenja11) [inline tutorials/howto-transfer-files-using-scp/01.en.md:9]:
```suggestion
author: "Your Name"
```

File: tutorials/howto-transfer-files-using-scp/01.en.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+(c) The contribution was provided directly to me by some other person
+    who certified (a), (b) or (c) and I have not modified it.
+
+(d) I understand and agree that this project and the contribution are
+    public and that a record of the contribution (including all personal
+    information I submit with it, including my sign-off) is maintained
+    indefinitely and may be redistributed consistent with this project
+    or the license(s) involved.
+
+Signed-off-by: Thomas Boehringer <dev@tboehringer.de>
```
Comment (svenja11) [inline tutorials/howto-transfer-files-using-scp/01.en.md:148]:
```suggestion
Signed-off-by: [submitter's name and email address here]
```

**Review by svenja11 — COMMENTED** [review]
Thank you for your contribution @pixia1234. Please double check the information I marked.

**Comment by pixia1234** [comment]
@svenja11 I have updated the required information. If there is anything else that needs to be changed, please feel free to contact me.

**Pull request merged**